domain/src/main/java/org/oppia/domain/oppialogger/loguploader/FirebaseLogUploader.kt

```diff
@@ -0,0 +1,39 @@
+package org.oppia.domain.oppialogger.loguploader
```

Comment:
Suggest putting this with the other Firebase deps to keep them together.

Reply:
Ditto as above.

Reply:
Done.